packages/ui-kit/src/components/PieChart/PieChart.module.scss

```diff
@@ -0,0 +1,87 @@
+@use '../../themes/typography';
+@use '../../themes/common';
+
+.rootPieChart {
+    @extend .typographyTiny;
+    overflow: auto;
+    padding: 0;
+
+    canvas {
+        height: var(--propel-component-height);
+    }
+}
+
+.pieChartTotalValue{
+    text-align: center;
+    margin-top: 10px;
+
+    span{ margin: 0; line-height: 22px; color:var(--propel-text-secondary)}
+    span:first-child{ font-size: 12px; padding-right: 5px; }
+    span:last-child{ font-size: 14px; color: var(--propel-text-primary)}
```

Comment:
I believe the Figma mock should have references to proper typography, e.g. `/* xSmall/Medium */`
Ideally, we should define font properties like:
```
span {
  @extend .typographyTiny;
  ...
}
```

or if more specific

```
span { line-height: var(--propel-tiny-line-height); }
```